### ISSUE TITLE
Add "does not exists anymore" for crossing quests

### ISSUE THREAD

**Issue by Codain** [streetcomplete/StreetComplete#3531]
Some pedestrian crossings disappear for whatever reason. One reason is if the crossing was a temporary crossing (yellow in France) during some construction works. Or it might be that it has been removed for safety reasons.

Proposal is to add a "Does not exists anymore" on at least the following crossing quests:
- crossing island yes/no
- crossing with strips for blind peoples yes/no
- crossing with sound for deaf peoples yes/no
- type of crossing markes/unmarked/lights

**Comment by matkoniecz** [comment]
Note that in such cases often much more significant editing needs to be done - at least for cases where there is separately mapped `highway=footway` `footway=crossing`

It maybe could be possible to add does not exists anymore for crossing not attached to `highway=path/footway/sidewalk/bridleway` but it would require

- significant effort to make this data available
- would be confusing as for many crossings it would not be shown

Leaving note with photo or fixing in Vespucci seems preferred solution.

**Comment by Codain** [comment]
Good point, it can be done at least on nodes not connected to `highway=footway` `footway=crossing` .

If connected to such way SC can maybe leave a note instead of removing tags?
